### DISCUSSION TITLE
Multi-browser and packaging problem

### DISCUSSION THREAD

**Discussion by c350147221** [seleniumbase/SeleniumBase#1238] [Q&A] [answered]
hi,
I want to re-encapsulate one floor through the BaseCase class, and you need to start multiple browsers, after the package __check _ scope method will throw it, ask this situation how to handle it?

**Comment by mdmintz** [comment, marked as answer]
If ``__check_scope()`` throws ``OutOfScopeException``, it means that you're not calling ``setUp()`` for the scope of your SeleniumBase methods. You can try following the syntax formats listed here to help you stay on track: https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/syntax_formats.md
If you need another driver in your test, call ``self.get_new_driver()``. Example test for that here: https://github.com/seleniumbase/SeleniumBase/blob/master/examples/test_multiple_drivers.py
